Rakefile

```diff
@@ -4,6 +4,7 @@ require 'rake/testtask'
 require 'yaml'
 require 'yajl'
 require 'open-uri'
+require 'json'
```

Comment:
The rake diff task fails w/o it, b/c it uses Hash.to_json. Apparently I was the first to use that in a while. Also see the separate commit message 7980c3d.
I could for sure separate that out into a separate PR if wanted, but it seems like a tiny fix on-the-go.
@lildude